consensus/hotstuff/pacemaker/timeout/config.go

```diff
@@ -18,9 +18,6 @@ type Config struct {
 	ReplicaTimeout float64
 	// MinReplicaTimeout is the minimum the timeout can decrease to [MILLISECONDS]
 	MinReplicaTimeout float64
```

Comment:
It will be updated as part of round duration research: https://github.com/dapperlabs/flow-go/issues/6203
But I will add a cap as you have suggested in this PR.